core/server/worker/src/test/java/alluxio/worker/block/stream/BlockWorkerDataReaderTest.java

```diff
@@ -151,7 +153,7 @@ public void createAndCloseManyReader() throws Exception {
       mBlockWorker.commitBlock(SESSION_ID, blockId, true);
       InStreamOptions inStreamOptions = new InStreamOptions(
           new URIStatus(new FileInfo().setBlockIds(Collections.singletonList(blockId))),
-          FileSystemOptions.openFileDefaults(mConf), mConf);
+          FileSystemOptions.openFileDefaults(mConf), mConf, null);
```

Comment:
The last argument is annotated with `NonNull`.

Reply:
Thank you, corrected it.